usaspending_api/common/helpers/endpoint_documentation.py

```diff
@@ -106,6 +105,7 @@ def validate_docs(url, url_object, master_endpoint_list):
         if re.fullmatch(pattern, endpoint):
             break
     else:
-        messages.append("No URL found in {} that matches {} ({})".format(ENDPOINTS_MD, url, qualified_name))
+        if qualified_name != "usaspending_api.common.views.RemovedEndpointView":
```

Comment:
This minor change bumped the code complexity past the set threshold. Perhaps adding the check before the loop would be cleaner